services/director-v2/src/simcore_service_director_v2/api/rpc/_computations.py

```diff
@@ -0,0 +1,74 @@
+# pylint: disable=too-many-arguments
+from fastapi import FastAPI
+from models_library.api_schemas_directorv2.comp_runs import (
+    ComputationRunRpcGetPage,
+    ComputationTaskRpcGetPage,
+)
+from models_library.products import ProductName
+from models_library.projects import ProjectID
+from models_library.rest_ordering import OrderBy
+from models_library.users import UserID
+from servicelib.rabbitmq import RPCRouter
+
+from ...modules.db.repositories.comp_runs import CompRunsRepository
+from ...modules.db.repositories.comp_tasks import CompTasksRepository
+from ..dependencies.database import get_repository_instance
```

Comment:
these dependencies are fastapi specifics for REST APIs. I would not use them here as this will create confusion.
Let's discuss tomorrow. there is already a built-in way to create the repository instances without it.